tests/pipelines/stable_diffusion/test_onnx_stable_diffusion.py

```diff
@@ -18,22 +18,39 @@
 import numpy as np
 
 from diffusers import OnnxStableDiffusionPipeline
-from diffusers.utils.testing_utils import require_onnxruntime, slow
+from diffusers.utils.testing_utils import is_onnx_available, require_onnxruntime, require_torch_gpu, slow
 
 from ...test_pipelines_onnx_common import OnnxPipelineTesterMixin
 
 
+if is_onnx_available():
+    import onnxruntime as ort
+
+
 class OnnxStableDiffusionPipelineFastTests(OnnxPipelineTesterMixin, unittest.TestCase):
     # FIXME: add fast tests
     pass
 
 
 @slow
 @require_onnxruntime
+@require_torch_gpu
 class OnnxStableDiffusionPipelineIntegrationTests(unittest.TestCase):
     def test_inference(self):
+        provider = (
+            "CUDAExecutionProvider",
+            {
+                "gpu_mem_limit": "17179869184",  # 16GB.
+                "arena_extend_strategy": "kSameAsRequested",
+            },
+        )
```

Comment:
Onnx tests now run with the CUDA provider. This enables us to add more integration tests without worrying about inference speed.